courses/admin.py

```diff
@@ -35,8 +35,8 @@ class CourseAdmin(admin.ModelAdmin):
     """Admin for Course"""
 
     model = Course
-    search_fields = ["title"]
-    list_filter = ["live", "program"]
+    search_fields = ["title", "topics__name"]
+    list_filter = ["live", "program", "topics__name"]
```

Comment:
Thanks but I believe, 
`list_filter = ["live", "program", "topics__name"]` will render with heading **By name** instead of **By topics** on right side of page for admin user.